.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: '🐜 Bug report'
+about: 'Report a reproducible bug.'
+title: ''
+labels: 'new-bug'
+---
+<!--
+NOTE: If this issue relates to security, please use the vulnerability disclosure form here:
+https://www.algorand.com/resources/blog/security
+
+General, developer or support questions concerning Algorand should be directed to the Algorand Forums https://forum.algorand.org/.
+-->
+
+### Subject of the issue
+Describe your issue here.
+
+### Your environment
+* Software version: `algod -v`
+* Node status if applicable: `goal node status`
+* Operating System details. 
+* In many cases log files and cadaver files are also useful to include, since these files may be large an Algorand developer may request them later.
```

Comment:
Nit: 

>In many cases log files and cadaver files are also useful to include*.* Since these files may be large*,* an Algorand developer may request them later.